Games using the built-in generated maps fail to start due to out-of-bounds points

Opening a game such as `spleef:standard` results in the error message "The game threw an unexpected error while starting!".

The following exception is printed to the log:

```
Caused by: java.lang.IllegalArgumentException: Point (16; 0) out of bounds Bounds[minX=-15, minZ=-15, maxX=15, maxZ=15]
	at xyz.nucleoid.spleef.game.map.shape.SpleefShape$Builder.index(SpleefShape.java:83) ~[main/:?]
	at xyz.nucleoid.spleef.game.map.shape.SpleefShape$Builder.putOutline(SpleefShape.java:77) ~[main/:?]
	at xyz.nucleoid.spleef.game.map.shape.ShapeCanvas.render(ShapeCanvas.java:40) ~[main/:?]
	at xyz.nucleoid.spleef.game.map.SpleefMapGenerator.build(SpleefMapGenerator.java:28) ~[main/:?]
```

--- a/src/main/java/xyz/nucleoid/spleef/game/map/shape/ShapeCanvas.java
+++ b/src/main/java/xyz/nucleoid/spleef/game/map/shape/ShapeCanvas.java
@@ -31,8 +31,8 @@ public final class ShapeCanvas {
         var shape = new SpleefShape.Builder(this.minX, this.minZ, this.maxX, this.maxZ);
         for (var entry : Long2BooleanMaps.fastIterable(this.points)) {
             long pos = entry.getLongKey();
-            int x = ChunkPos.getPackedX(pos) - this.minX;
-            int z = ChunkPos.getPackedZ(pos) - this.minZ;
+            int x = ChunkPos.getPackedX(pos);
+            int z = ChunkPos.getPackedZ(pos);
 
             if (entry.getBooleanValue()) {
                 shape.putFill(x, z);

